Lesson_4/README.md

```diff
@@ -0,0 +1 @@
+# redi_js_spring_2020
```

Comment:
What is the purpose of this README file? If you don't plan to fill it in with some explanation you can remove it.